pooch/core.py

```diff
@@ -369,3 +369,26 @@ def load_registry(self, fname):
                     file_url = elements[2]
                     self.urls[file_name] = file_url
                 self.registry[file_name] = file_sha256
+
+    def check_availability(self, fname):
```

Comment:
```suggestion
    def is_available(self, fname):
```

Since this method returns True/False, it's better to read `if p.is_available('my_file.csv')`.